examples/decomposition/plot_varimax_fa.py

```diff
@@ -0,0 +1,80 @@
+"""
+===============================================================
+Factor Analysis (with rotation) to visualize patterns
+===============================================================
+
+Investigating the Iris dataset, we see that sepal length, petal
+length and petal width are highly correlated. Sepal width is
+less redundant. Matrix decomposition techniques can uncover
+these latent patterns. Applying rotations to the resulting
+components does not inherently improve the predictve value
+of the derived latent space, but can help visualise their
+structure; here, for example, the varimax rotation, which
+is found by maximizing the squared variances of the weights,
+finds a structure where the second component only loads
+positively on sepal width.
+"""
+
+# Authors: Jona Sassenhagen
+# License: BSD 3 clause
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn.decomposition import FactorAnalysis, PCA
+from sklearn.preprocessing import StandardScaler
+from sklearn.datasets import load_iris
+
+print(__doc__)
+
+##############################################################################
```

Comment:
We are now using the following for section dividers:
```suggestion
# %%
```

Reply:
Done